### ISSUE TITLE
Loss function used

### ISSUE THREAD

**Issue by poganyg** [Docea/Kaggle_HaemorrhageDetection#5]
Apparently (https://www.youtube.com/watch?v=IVVVjBSk9N0) L1Loss is usually used for regression problems and not classification ones. I am looking into cross entropy losses where we can use a weight tensor for penalising the dominant classes.
